toolchain/sem_ir/interface.h

```diff
@@ -42,6 +48,8 @@ struct Interface : public Printable<Interface> {
 
   // The following members are set at the `}` of the interface definition.
   bool defined = false;
+  // The declarations of the associated entities.
+  InstBlockId associated_entities_id = InstBlockId::Invalid;
```

Comment:
Is this always set? If so, can `defined` be removed? If not, maybe the comment should note it?

Reply:
Sure, done.